scss/_utilities_hide.scss

```diff
@@ -1,5 +1,14 @@
 @import 'settings';
 
+// mixin for hiding table columns
+@mixin vf-u-hide-table-column {
```

Comment:
Is this better as part of the hide utilities or a table pattern utility? This is scoped to table and so would expect to be able to use it if I import the table pattern.

Reply:
True. I thought of it as modification of `u-hide` pattern (especially that it shares the `--small`, etc variants), so I kept it here.
It would especially make sense if we decide to do it as `u-hide` scoped to the table in the end.

If we keep it as new utility it may make sense to separate it, although, I don't think we have any other example of utility bound with a pattern, so I'd probably just separate it to different file.

Reply:
If a utility is bound so tightly to a pattern then it might not be a utility at all but a pattern element. This might be much nicer as a state class on table cells. For example: `is-hidden--small`?

Reply:
This would be much easier if we used classes on each element but tables are difficult in this case.

Reply:
Or even doubling down on a11y and targeting just
```css
&[aria-hidden='true'] {
  ...
}
```

Reply:
We can make `aria-hidden` replace `u-hide-table-column` (or even `u-hide`), but this will not work with breakpoints. We still need a classname to put `--small`, etc variants on.

Using table specific styling is indeed tempting as these are table specific styles. The semantic problem is that we don't really have "table pattern" (`p-table`), so we don't have a class names to modify.
This leads us to modifier classes like `is-hidden--small` you suggested. But this class name by itself doesn't tell it's table specific. It could be easily mistaken for `u-hide--small`. So technically we should do it as `p-table__cell--hidden--small` or something along that lines - which get's weirdly long and specific, but on the other hand does exactly what it says, where it says.